dask/compatibility.py

```diff
@@ -15,10 +15,10 @@ def entry_points(group=None):
     In 3.10 the return type changed from a dict to an ``importlib.metadata.EntryPoints``.
     This compatibility utility can be removed once Python 3.10 is the minimum.
     """
-    eps = importlib.metadata.entry_points()
-    if group:
-        try:
-            return eps.select(group=group)
-        except AttributeError:
+    if _PY_VERSION >= parse_version("3.10"):
+        return importlib.metadata.entry_points(group=group)
+    else:
+        eps = importlib.metadata.entry_points()
+        if group:
             return eps.get(group, [])
-    return eps
+        return eps
```

Comment:
This is purely just moving code around to we can use `_PY_VERSION` instead of a `try`/`except`, not a functional change. IMO this makes it a bit easier to see when this can be removed as we'll grep for `_PY_VERSION` when adding/removing Python versions. Just personal preference -- happy to discard this change if others feel strongly

Reply:
> this makes it a bit easier to see when this can be removed as we'll grep for _PY_VERSION when adding/removing Python versions

makes sense to me!